lib/CL/clSetKernelExecInfo.c

```diff
@@ -111,9 +123,11 @@ POname(clSetKernelExecInfo)(cl_kernel kernel,
         for (size_t i = 0; i < param_value_size / sizeof (void *); ++i)
           {
             void *dev_ptr = ptrs[i];
-            pocl_raw_ptr *raw_ptr_info
-                = pocl_find_raw_ptr_with_dev_ptr (kernel->context, dev_ptr);
-            POCL_RETURN_ERROR_ON ((raw_ptr_info == NULL), CL_INVALID_VALUE,
+            // TODO: cl_ext_buffer_device_address 1.0.1 allows invalid
+            // pointers in clSetKernelArgDevicePointerEXT - does it
+            // apply on CL_KERNEL_EXEC_INFO_DEVICE_PTRS_EXT too?
```

Comment:
This is not specified to my recollection. I think it's best to return invalid value here. Allowing it as kernel arguments is justified with C/C++ which allow passing garbage pointers around until they are dereferenced.

Reply:
Also please use the old /* */ style in C comments.